picard/formats/vorbis.py

```diff
@@ -49,6 +49,13 @@
 )
 
 
+INVALID_CHARS = re.compile('([^\x20-}]|=)')
```

Comment:
What is the source for this list of invalid characters ?
Also please add specific tests for it (in addition of format's one i mean).

Reply:
This is from the Vorbis specs at https://www.xiph.org/vorbis/doc/v-comment.html#vectorformat and is the reverse of what is tested at https://github.com/quodlibet/mutagen/blob/master/mutagen/_vorbis.py#L26

I added some docstring and a test. Regarding the test I think `test_formats.py` is getting too crowded. I think we should split it into separate files per format (with the common test cases kept at a central file), but that's something for another PR.

Reply:
See commit https://github.com/metabrainz/picard/pull/1150/commits/cf9528bd86d3cd49eecc6c7ffcb49c63776ade52

Reply:
>  I think we should split it into separate files per format (with the common test cases kept at a central file), but that's something for another PR.

Yes good idea !